### PR TITLE
Prerelease 2.6.3

### PR DESCRIPTION
**Pull request by rlmv**
# Description

Fix Travis release step (again)

## Github Issue

[ISSUE-NUMBER](issue-url)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added unit tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -41,17 +40,20 @@ jobs:
 
     - stage: release
       if: tag IS present
+      python: '3.6'
       install: skip
       script: skip
       deploy:
         - provider: pypi
-          on: master
+          on:
```
Comment (mattusifer) [inline .travis.yml]: can we move this `on:` into the `stage: release` section instead of repeating it for each `deploy` section?

Comment (rlmv) [reply .travis.yml]: No, I tried but it doesn't seem to work like that, alas

Comment (rlmv) [reply .travis.yml]: I don't know if that's a bug or intended behavior

Comment (mattusifer) [reply .travis.yml]: ah alright, no worries then

**Pull request merged**